src/psyclone/psyir/symbols/containersymbol.py

```diff
@@ -163,6 +169,27 @@ def wildcard_import(self, value):
                             f"'{type(value).__name__}'")
         self._has_wildcard_import = value
 
+    @property
+    def intrinsic(self):
+        '''
+        :returns: whether or not this module is an intrinsic module.
+        :rtype: bool
+        '''
+        return self._intrinsic
+
+    @intrinsic.setter
+    def intrinsic(self, value):
+        '''
+        Set whether or not this is an intrinsic module import.
+        :param bool value: whether or not this is an intrinsic module.
+
+        :raises TypeError: if the supplied `value` is not a bool.
+        '''
+        if not isinstance(value, bool):
+            raise TypeError(f"intrinsic must be a bool but got: "
+                            f"'{type(value).__name__}'")
```

Comment:
Start with capital letter and end with punctuation.

Reply:
Added